### ISSUE TITLE
SSG E170

### ISSUE THREAD

**Issue by muchimi** [skiselkov/BetterPushbackC#153]
Hi,
couple of issues related to the SSG E170, that includes its own pushback mechanism which is probably why it's so challenging to use betterpushback with it.

(1) hydraulics have to be turned off or the plane will not move and better pushback gets "stuck".  Even when off, the pushback has a strange initial "kink" in the path with the tug going almost to a 45 degree angle before going straight again (even if the path is straight). 

(2) nosewheel steering needs to be enabled (which is ok but won't work if the hydraulics are off).  

To get the pushback to work, the first thing to do is to enable nosewheel steering, then turn the hydraulics off.   That means you can't really start engines or do anything with the plane during the pushback itself as having no hydraulics means nothing really works as intended until they are enabled again.  Once the tug is disconnected, then enable hydraulics and go through startup.  

In other words, checklists has to pause during pushback.

This is a similar issue with the SSG 747.

I think it will require some workaround code to override the SSG logic.

**Comment by skiselkov** [comment]
Um, ok, but this seems more like a bug that you need to file with SSG. If the aircraft keeps its brakes on or messes with the nosewheel steering, then there's not much I can do about that. Is there something you think I could fix? (Please note: I don't own the SSG E170 or 747).

**Comment by muchimi** [comment]
I can post a support ticket in the SSG line and see what they say.

**Comment by skiselkov** [comment]
Closing as inactive.
